.contrib/Parser/DATAS/02 - Outdoor Zones/05 Pandaria/Krasarang Wilds.lua

```diff
@@ -751,6 +772,9 @@ root(ROOTS.Zones, {
 										["provider"] = { "n", 68322 },	-- Muerta
 									}),
 								}),
+								ach(8205, {		-- Operation: Shieldwall
```

Comment:
This one has not crit?

Reply:
That one is just for reaching exalted rep, so no crit.